pandas/core/nanops.py

```diff
@@ -129,7 +132,7 @@ def f(values, axis=None, skipna=True, **kwds):
         return f
 
 
-def _bn_ok_dtype(dt, name: str) -> bool:
+def _bn_ok_dtype(dt: Dtype, name: str) -> bool:
```

Comment:
can you dt -> dtype

Reply:
Addressed in https://github.com/pandas-dev/pandas/pull/30461/commits/328356467eb6d4b636a2e5acf696b059648ce5b5